app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True, editable=False)
```

Comment:
<img width="667" alt="screen shot 2016-09-26 at 11 25 08 am" src="https://cloud.githubusercontent.com/assets/12392709/18831123/f56b4990-83db-11e6-9142-42195bbb9bd5.png">

I think it's there for clarification of the cause of error when name uniqueness is violated. @andela-kanyanwu: Am I right?